### ISSUE TITLE
Specifying a container for lightGallery

### ISSUE THREAD

**Issue by tpaksu** [sachinchoolur/lightGallery#421]
Hi,

I have a full page layout with a top menu, a sidebar and a content area. I want the lightGallery have the bounds of the content area, so the sidebar and the top menu will still be visible. Is it possible?

**Comment by UksusoFF** [comment]
Please describe solution if you found this.

**Comment by tpaksu** [comment]
It's an old topic but I guess I've changed the z-index of the header element to show on top of the lightGallery, and gave the body element some margin. For example;

1. The lightGallery overlay (lg-backdrop) has a z-index of 1040, I've set the header (navbar-row) z-index to 2000. 
2. my header (navbar-row) has a height of 50px; I gave the body element a 50px top margin, so the lightGallery would also have a top margin. and the header would be over the lightGallery.

**Note**: header should have an absolute position and body should not have a relative position defined to make this work.
